src/main/java/com/google/api/codegen/transformer/php/PhpGapicSurfaceTestTransformer.java

```diff
@@ -61,32 +65,71 @@
   private final TestCaseTransformer testCaseTransformer = new TestCaseTransformer(valueProducer);
   private final MockServiceTransformer mockServiceTransformer = new MockServiceTransformer();
   private final PhpFeatureConfig featureConfig = new PhpFeatureConfig();
+  private final PhpGapicCodePathMapper pathMapper =
+      PhpGapicCodePathMapper.newBuilder().setPrefix("tests/unit").build();
 
   @Override
   public List<String> getTemplateFileNames() {
-    return Collections.singletonList(TEST_TEMPLATE_FILE);
+    return ImmutableList.<String>of(TEST_TEMPLATE_FILE, MOCK_SERVICE_TEMPLATE_FILE);
   }
 
   @Override
   public List<ViewModel> transform(Model model, ApiConfig apiConfig) {
     List<ViewModel> models = new ArrayList<ViewModel>();
     PhpSurfaceNamer namer = new PhpSurfaceNamer(apiConfig.getPackageName());
```

Comment:
Since you have a `testPackageNamer` you may rename this into something like `surfacePackageNamer`.